src/DiscList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { getDiscs } from './fetch-utils';
+import './DiscList.css'
+class DiscList extends Component {
+    state = { discs: [] };
+    componentDidMount = async () => {
+        const data = await getDiscs();
+        this.setState({ discs: data });
+    };    
+    render() { 
+        return (
+            <section className='disc-list'>
+                {this.state.discs.map((d) => (
+                    <div key={d.id} className='disc-card'>
```

Comment:
Since this area is ultimately a list of items, I'd like to see you try to use the UL/LI syntax more instead of `<div>` tags. You could alternatively use the `<article>` tag within the section and that would accomplish both semantics and accessibility.